cucumber/authenticators_gcp/features/step_definitions/authn_gcp_steps.rb

```diff
@@ -0,0 +1,31 @@
+Given(/^I set (authn-gcp\/service-account-id|authn-gcp\/service-account-email|authn-gcp\/project-id|authn-gcp\/instance-name) annotation (with incorrect value )?to host "([^"]*)"$/) do |annotation_name, incorrect_value, hostname|
```

Comment:
you can change the regex to `^I set authn-gcp\/(service-account-id|service-account-email|project-id|instance-name) annotation (with incorrect value )?to host "([^"]*)"`

this way the `authn-gcp/` prefix is not duplicated